cpp/include/cudf/io/orc.hpp

```diff
@@ -197,10 +197,9 @@ class orc_reader_options {
    * @throw cudf::logic_error if a negative value is passed
    * @throw cudf::logic_error if stripes have been previously set
    */
-  void set_skip_rows(uint64_t rows)
+  void set_skip_rows(int64_t rows)
   {
     CUDF_EXPECTS(rows == 0 or _stripes.empty(), "Can't set both skip_rows along with stripes");
-    CUDF_EXPECTS(rows <= std::numeric_limits<int64_t>::max(), "skip_rows is too large");
```

Comment:
Actually we can skip very large number of rows.

Reply:
the new API has exactly the same limit

Reply:
We no longer have any limit on the skip row.

Reply:
It's limited by the option type, which is int64_t with your changes. The removed line here also limits the value to what int64_t can hold. There's no change in the limit.

Reply:
Ah I see that.